analyzer/queries.go

```diff
@@ -508,6 +519,48 @@ func (qf QueryFactory) RuntimeEVMTokenUpdateQuery() string {
 			token_address = $1`, qf.chainID, qf.runtime)
 }
 
+func (qf QueryFactory) RuntimeEVMTokenBalanceAnalysisStaleQuery() string {
+	return fmt.Sprintf(`
+		SELECT
+			evm_token_balance_analysis.runtime,
+			evm_token_balance_analysis.token_address,
+			evm_token_balance_analysis.account_address,
+			evm_token_balance_analysis.last_mutate_round,
+			evm_tokens.token_type,
+			evm_token_balances.balance,
+			token_address_preimage.context_identifier,
+			token_address_preimage.context_version,
+			token_address_preimage.address_data,
+			account_address_preimage.context_identifier,
+			account_address_preimage.context_version,
+			account_address_preimage.address_data
+		FROM %[1]s.evm_token_balance_analysis
+		LEFT JOIN %[1]s.evm_tokens USING (runtime, token_address)
```

Comment:
let's make this INNER JOIN so we don't have to bail on unscanned tokens from the go side

Reply:
probably ditto for all the other JOINs in this query?

Reply:
:slightly_smiling_face:

Reply:
foiled again. the criteria is supposed to be evm_token_analysis.last_download_round IS NOT NULL instead of evm_tokens.token_type NULL or row missing :facepalm:

Reply:
updated, gonna try running some EXPLAIN tomorrow to make sure the query plan doesn't go crazy. still using LEFT JOIN for most things though :pirate_flag: